test/cases/cleanups/nonTypeErrors/expected.ts

```diff
@@ -0,0 +1,6 @@
+// @ts-expect-error -- TODO: This module can only be default-imported using the 'esModuleInterop' flag
+import ts from "typescript";
+// @ts-expect-error -- TODO: Cannot find module './typestat.json'. Consider using '--resolveJsonModule' to import module with '.json' extension.
```

Comment:
[Testing] The test should have `resolveJsonModule` so this gets imported nicely.